boa/coverage.py

```diff
@@ -68,7 +68,7 @@ def dynamic_source_filename(self, filename, frame):
         if contract is None:
             return None
 
-        return str(contract.filename)
+        return str(contract.filename) if contract.filename else None
```

Comment:
maybe change line 68 to: `if contract is None or contract.filename is None`

Reply:
@charles-cooper done ✅